src/main/java/com/mathworks/ci/MatlabReleaseInfo.java

```diff
@@ -112,4 +99,56 @@ else if(!new File(this.matlabRoot).exists()){
         }
         return versionInfoCache;
     }
+    
+    /*
+     * Static File Callable to perform File operations on specific remote nodes.
+     */
+        
+    private static final class RemoteFileOperation implements FileCallable<Map<String, String>> {
+
+        private static final long serialVersionUID = 1L;
+
+        private Map<String, String> versionInfoCache = new HashMap<String, String>();
+
+        @Override
+        public void checkRoles(RoleChecker checker) throws SecurityException {
+
```

Comment:
If this is intended to be empty, consider adding a comment saying so. Maybe: `// no checks to perform`